src/main/java/io/reactivex/Flowable.java

```diff
@@ -191,6 +194,9 @@ public static int bufferSize() {
      * If any of the sources never produces an item but only terminates (normally or with an error), the
      * resulting sequence terminates immediately (normally or with all the errors accumulated till that point).
      * If that input source is also synchronous, other sources after it will not be subscribed to.
+     * <p>
+     * If there are no source Publishers provided, resulting sequence completes immediately without emitting
```

Comment:
"If there are no source Publishers provided, the resulting sequence completes immediately without emitting any items and without any calls to the combiner function."